internal/engine/bundle_builder.go

```diff
@@ -160,11 +161,7 @@ func (b *BundleBuilder) Build() (cue.Value, error) {
 func (b *BundleBuilder) getInstanceUrl(v cue.Value) string {
 	url, _ := v.String()
 	if path := strings.TrimPrefix(url, apiv1.LocalPrefix); IsFileUrl(url) && !filepath.IsAbs(path) {
-		source := v.Pos().Filename()
-		if origin, ok := b.mapSourceToOrigin[source]; ok {
-			source = origin
-		}
-		url = apiv1.LocalPrefix + filepath.Clean(filepath.Join(filepath.Dir(source), path))
+		url = apiv1.LocalPrefix + filepath.Clean(filepath.Join(b.cwd, path))
```

Comment:
We can't use CWD here, the relative path is to the bundle file (what ` v.Pos().Filename()` does) instead of the working dir of the CLI. You can see the e2e tests failing for: 

```
timoni bundle build -f hack/tests/nginx_bundle.cue --runtime-from-env

ERR module not found at path /home/runner/work/blueprints/starter
```
```

Reply:
> We can't use CWD here, the relative path is to the bundle file (what  v.Pos().Filename() does) instead of the working dir of the CLI.

Ha ok, I understand the logic better now, thanks.

This raises the issue that when the bundle file comes from Stdin (`-f -`), the `v.Pos().Filename()` becomes the temporary directory of the file Stdin was written to. Which makes the `getInstanceUrl` function fail again, since the path is now wrong.

I'm also unclear what will happen when someone specifies both a bundle via Stdin and a file (`-f- -f bundle.cue`).

I'll look into that.

Reply:
I think we can error out if the bundle comes from stdin and contains local refs.

Reply:
Sounds good.

I removed the `cwd` code and added 4 new tests:
* bundle apply with file from stdin (`-f-`) and `module: url: file://./relative/path/to/module`
* bundle apply with file from stdin (`-f-`) and `module: url: file:///absolute/path/to/module`
* bundle apply with file from concrete file (`-f /path/to/bundle.cue`) and `module: url: file://./relative/path/to/module`
* bundle apply with file from concrete file (`-f /path/to/bundle.cue`) and `module: url: file:///absolute/path/to/module`

How's that?